pkg/daemon/ceph/client/command.go

```diff
@@ -33,8 +33,8 @@ import (
 var RunAllCephCommandsInToolboxPod string
 
 const (
-	// AdminUsername is the name of the admin user
-	AdminUsername = "client.admin"
+	// AdminUsername is the name of the operator admin user
+	AdminUsername = "client.rookoperator"
```

Comment:
How about we distinguish this as the operator vs non-operator keyrings
```suggestion
	OperatorAdminUsername = "client.rookoperator"
```